etna/collections/migrations/0022_rename_record_iaid_resultspagerecord_record_metadataid.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.13 on 2022-07-19 10:15
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("collections", "0021_add_feature_insights"),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name="resultspagerecord",
+            old_name="record_metadataId",
```

Comment:
Something weird has gone on here. This should be:

```suggestion
            old_name="record_iaid",
```